release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1990,6 +1990,10 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
 
 * Previously, if you created a pipeline with parameters that had empty strings, for example, `”`, the fields in the {product-title} web console would not accept the empty strings. The current release fixes this issue. Now, `”` is supported as a valid default property within the parameters section. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951043[*BZ#1951043*])
 
+* Previously, the users were not able to create a knative service as a cluster local from the *Developer* perspective. This issue has now been fixed by changing the label required for making a service cluster local to `'networking.knative.dev/visibility': 'cluster-local'`. The users are now able to create a knative service as cluster local from the *Developer* perspective. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970796[*BZ#1970796*])
+
+* Previously, the Kamelets of type sink were shown in the catalog for event sources along with the type source. This issue has now been fixed by filtering Kamelets resources to list only the source type. Now Kamelets of only type source would be displayed in the catalog for event sources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972258[*BZ#1972258*])
```

Comment:
```suggestion
* Previously, the Kamelets of type sink were shown in the catalog for event sources along with the type source. This issue has now been fixed by filtering Kamelets resources to list only the source type. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972258[*BZ#1972258*])
```

I think you can remove the last line since it's just sort of repeating what was stated in the previous line.
RE the other bullet point, see the feedback from the 4.6 PR review, since that applies here also.

Reply:
Thanks a lot for the feedback.